### PR TITLE
Updating the README to include Installation Problems and Resolutions

### PR DESCRIPTION
**Pull request by oakeyc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,9 @@
    $ pip install --user azure-cli-shell
 ```
 
+A list of common install issues and their resolutions are available at [install troubleshooting](https://github.com/Azure/azure-cli-shell/blob/master/doc/install_troubleshooting.md).
```
Comment (derekbekoe) [inline README.md]: Think this url would be wrong.
The file below looks like the directory is docs.

**Comment by oakeyc** [comment]
changes made

**Comment by derekbekoe** [comment]
@oakeyc The url points to the wrong repo

**Comment by oakeyc** [comment]
changes made

**Review by derekbekoe — APPROVED** [review]

**Pull request merged**